### PR TITLE
Add github ci.

### PR DESCRIPTION
**Pull request by Pfeil**
- should we remove the travis.yml?
- what has to be done for $code_coverage_dashboard?

### CODE REVIEW

**Comment by ThomasJejkal** [comment]
Point 1:  yes
Point 2:  Doesn't the [metastore2 configuration](https://github.com/kit-data-manager/metastore2/blob/master/.github/workflows/gradle.yml) helps on this?

**Comment by Pfeil** [comment]
Point 1: Done
Point2: Well yes and no. Metastore (and other projects, too) also have a gradle dependency to a coveralls library, and I am not sure yet what the official way is etc.

**Comment by ThomasJejkal** [comment]
Well, I think you need the dependency anyways. Which alternative do you see?

**Comment by Pfeil** [comment]
I think the dependency is not used by the CI, is it? Maybe I just do not understand how it works. I also just noticed that the metastore config uses codecov and I am talking about the coveralls dependency. Do we have a decision on what to use?

**Comment by Pfeil** [comment]
I added the codecov CI workflow. Can be merged now. I will remove the coveralls dependency in the gradle file in #6 .

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Set up OpenJDK version ...
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'zulu' 
+        java-version: ${{ env.currentBuildVersion }}
```
Comment (VolkerHartmann) [inline .github/workflows/gradle.yml:55]: You have to define this environment variable at the top of this file.

**Review by VolkerHartmann — CHANGES_REQUESTED** [review]
See comment...

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (Pfeil) [reply .github/workflows/gradle.yml]: Whoops. Done. I renamed it a little so it is hopefully clear what part it influences.

**Review by VolkerHartmann — APPROVED** [review]
Well done!

**Pull request merged**